src/pveclib/vec_char_ppc.h

```diff
@@ -753,6 +753,41 @@ vec_popcntb (vui8_t vra)
 #define vec_popcntb __builtin_vec_vpopcntb
 #endif
 
+/*! \brief Vector Set Bool from Signed Byte.
+ *
+ *  For each byte, propagate the sign bit, to all 8-bits of that
```

Comment:
s/propagate the sign bit, to/propagate the sign bit to/
this is a global comment, as there are more instances of this below...

Reply:
OK